layouts/shortcodes/observability_pipelines/configure_existing_pipelines/destination_env_vars/sumo_logic.md

```diff
@@ -0,0 +1,4 @@
+- Unique URL generated for HTTP Logs and Metrics source to receive log data 
```

Comment:
```suggestion
- Unique URL generated for the HTTP Logs and Metrics source to receive log data 
```
Just guessing, it seemed like a word was missing here, ignore if not!

Reply:
Edited suggestion to capitalize "Source" because that's how Sumo Logic does it,